troposphere/static/js/components/common/breadcrumb/Breadcrumb.react.js

```diff
@@ -1,42 +1,44 @@
-define(function (require) {
+define(function(require) {
 
   var React = require('react/addons'),
     Router = require('react-router');
 
   return React.createClass({
 
+    displayName: "Breadcrumb",
+
     propTypes: {
       breadcrumb: React.PropTypes.object.isRequired,
       onClick: React.PropTypes.func.isRequired,
+      breadcrumbText :React.PropTypes.string.isRequired,
     },
 
-    mouseOver: function () {
+    mouseOver: function() {
       this.props.onMouseOn(this.props.breadcrumb.name);
     },
 
-    mouseOut: function () {
+    mouseOut: function() {
       this.props.onMouseOff();
     },
 
-    renderLink: function () {
+    renderLink: function() {
       var class_names = this.props.breadcrumb.state;
       var divStyle = {
         width: this.props.width + '%'
       };
 
       return (
-        <div style={divStyle} onMouseOver={this.mouseOver} onMouseOut={this.mouseOut} className={class_names}
-             onClick={this.crumbClicked}>
-          {this.props.breadcrumb.step + 1}
-        </div>
-      );
+          <div /* style={divStyle} */ onMouseOver={this.mouseOver} onMouseOut={this.mouseOut} className={class_names} onClick={this.crumbClicked}>
```

Comment:
Is there a reason we commented this out? If its not necessary lets remove it altogether.